Amplify/Amplify.swift

```diff
@@ -76,6 +81,24 @@ public class Amplify {
                 "Verify that the library version is correct and supports the plugin's category.")
         }
     }
+
+    static var isTesting: Bool {
+        ProcessInfo.processInfo.environment["XCTestConfigurationFilePath"] != nil
```

Comment:
Is the key "XCTestConfigurationFilePath" going to change later on? Or maybe can this be replaced with a swift library constant?

Reply:
this is the most common way that code checks if unit tests are being run. if this does change we can update the code with that change.